### PR TITLE
SUPPORTENG-986 Add self-serve Collaborator

### PR DESCRIPTION
**Pull request by cameronjp**
https://zapierorg.atlassian.net/browse/SUPPORTENG-986

### CODE REVIEW

File: docs/_manage/account-and-team/add-team.md [hunk 1]
```diff
@@ -8,33 +8,63 @@ redirect_from:
     - /manage/invite-team-member
 ---
 
-# Add team members to your integration
+# Invite Team Members to Your Integration
```
Comment (marinahand) [inline docs/_manage/account-and-team/add-team.md:11]: Invite team members to your integration

File: docs/_manage/account-and-team/add-team.md [hunk 2]
```diff
@@ -8,33 +8,63 @@ redirect_from:
...
 
-- **Collaborator**: Collaborators have view-only access to your Zapier integration (including integration details, history, and analytics). Collaborators are unable to edit but can view performance data, view the integration history, comment on feature requests and bugs, and access tools to embed Zapier integrations inside your UI to rack up user adoption. Collaborator access is recommended for product, leadership, partnership, marketing, and other teams that do not contribute directly to the creation and maintenance of the integration, but need to get data regarding the integration for analysis and record purposes.
+- **Collaborator**: Collaborators are granted read-only access to the integration. Collaborator access is recommended for product, leadership, partnership, marketing, and other teams seeking access to integration data for analysis and record-keeping without direct involvement in its creation and maintenance. While they cannot make direct changes to the integration, they can:
 
-## Adding team members
```
Comment (marinahand) [inline docs/_manage/account-and-team/add-team.md:19]: Add team members

File: docs/_manage/account-and-team/add-team.md [hunk 3]
```diff
@@ -8,33 +8,63 @@ redirect_from:
...
+- Access tools to embed your integration throughout your site to drive user adoption
 
-To add team members to your integration:
-1. Log into the [Platform UI](https://zapier.com/app/developer).
-2. Select your **integration**. 
-3. In the _Manage_ section in the left sidebar, click on **Manage Team**.
-4. Click on the **Invite Team Member** button.
-5. Enter the team member's email address to be invited.
-6. Select the **Team Member Role**.
+## Inviting Team Members
```
Comment (marinahand) [inline docs/_manage/account-and-team/add-team.md:24]: Invite team members

File: docs/_manage/account-and-team/add-team.md [hunk 4]
```diff
@@ -8,33 +8,63 @@ redirect_from:
...
+8. Click “Send Invite”
 
-When you do this, an invitation email would be sent to the invitee, asking them to accept the invitation.
+> **Note**: Admins can invite both Admins and Collaborators, while Collaborators can only invite Collaborators.
 
-Once an invitee accepts the invite, they can access the integration through their own Zapier account. If the invitee does not have a Zapier account, accepting the invite would prompt them to sign up one first before they can access the integration.
+Once you click "Send Invite", an email invitation will be sent to the invitee, requesting their acceptance. If the invitee does not already have a Zapier account associated with the invited email, they will need to create an account first. Upon accepting the invitation, they will gain access to the integration through their Zapier account on the Developer Platform.
 
-## View and remove team members
+## Self-Serve Collaborator Access
```
Comment (marinahand) [inline docs/_manage/account-and-team/add-team.md:41]: Self-serve collaborator access

File: docs/_manage/account-and-team/add-team.md [hunk 5]
```diff
@@ -8,33 +8,63 @@ redirect_from:
...
+5. Click “Edit email domains”
+6. Add/remove the domain(s) to be allowed
+
+### Joining an Integration Team via Self-Serve Collaborator Access
+1. Log into the [Developer Platform](https://zapier.com/app/developer)
+2. Click “Get Access”
+3. Search for and select your integration, then click “Submit”
+4. If the Zapier account you're using is associated with an approved domain, you'll receive an invitation to join via email
+
+## View and Remove Team Members
```
Comment (marinahand) [inline docs/_manage/account-and-team/add-team.md:68]: View and remove team members

File: docs/_manage/account-and-team/add-team.md [hunk 6]
```diff
@@ -8,33 +8,63 @@ redirect_from:
...
+
+### Managing Eligible Email Domains for Self-Serve Collaborator Access
+1. Log into the [Developer Platform](https://zapier.com/app/developer)
+2. Choose your integration
+3. In the "Manage" section on the left sidebar, click "Manage Team"
+4. Click “Settings”
+5. Click “Edit email domains”
+6. Add/remove the domain(s) to be allowed
+
+### Joining an Integration Team via Self-Serve Collaborator Access
```
Comment (marinahand) [inline docs/_manage/account-and-team/add-team.md:62]: Join an integration team via self-serve collaborator access

File: docs/_manage/account-and-team/add-team.md [hunk 7]
```diff
@@ -8,33 +8,63 @@ redirect_from:
...
+### Enabling or Disabling Self-Serve Collaborator Access
+
+To toggle the “Self-serve join as Collaborator” feature, please follow these steps:
+1. Log into the [Developer Platform](https://zapier.com/app/developer)
+2. Choose your integration
+3. In the "Manage" section on the left sidebar, click "Manage Team"
+4. Click “Settings”
+5. Click the toggle to enable or disable self-serve join as Collaborator
+
+### Managing Eligible Email Domains for Self-Serve Collaborator Access
```
Comment (marinahand) [inline docs/_manage/account-and-team/add-team.md:54]: Manage eligible email domains for self-serve collaborator access

File: docs/_manage/account-and-team/add-team.md [hunk 8]
```diff
@@ -8,33 +8,63 @@ redirect_from:
...
 
-## View and remove team members
+## Self-Serve Collaborator Access
 
-You can view a list of invited team members (both those that have accepted the invites and those with pending invites) from the **Manage Team** page. An admin can also remove other team members from this page. However, please note that if you are removed as an Admin, you will only be able to regain Admin access if invited by an existing Admin.
\ No newline at end of file
+Depending on your [integration settings](https://cdn.zappy.app/b2637f3ad910c36c4e8c8224c349beee.png), users have the option to self-serve and join an integration team as Collaborators. This feature employs domain-name verification and is managed by integration Admins. Currently, this feature is only available for integrations listed in [Zapier's directory](https://zapier.com/apps).
+
+### Enabling or Disabling Self-Serve Collaborator Access
```
Comment (marinahand) [inline docs/_manage/account-and-team/add-team.md:45]: Enable or disable self-serve collaborator access

**Review by marinahand — CHANGES_REQUESTED** [review]
Thanks for the rewrite @cameronjp! Made some minor comments to keep to the style guide we've been directed to follow for these docs 😊
https://coda.io/d/Zapier-Content-and-Messaging-Guide_dH5LiS8Mxyt/Style-and-mechanics_suRp5#_ludID

File: docs/_manage/account-and-team/add-team.md (hunk 1 above)
Comment (cameronjp) [reply docs/_manage/account-and-team/add-team.md]: Fixed and pushed in new push!

File: docs/_manage/account-and-team/add-team.md (hunk 2 above)
Comment (cameronjp) [reply docs/_manage/account-and-team/add-team.md:19]: This was taken out, nothing to update here

File: docs/_manage/account-and-team/add-team.md (hunk 3 above)
Comment (cameronjp) [reply docs/_manage/account-and-team/add-team.md]: Fixed and pushed in new push!

File: docs/_manage/account-and-team/add-team.md (hunk 5 above)
Comment (cameronjp) [reply docs/_manage/account-and-team/add-team.md]: Fixed and pushed in new push!

File: docs/_manage/account-and-team/add-team.md (hunk 7 above)
Comment (cameronjp) [reply docs/_manage/account-and-team/add-team.md]: Fixed, but I kept the capital "C" for Collaborator due to how it's being used as a verb. Collaborator is capitalized everywhere else, shouldn't it also be capitalized here? Submitted new push.

File: docs/_manage/account-and-team/add-team.md (hunk 8 above)
Comment (cameronjp) [reply docs/_manage/account-and-team/add-team.md]: Fixed, but I kept the capital "C" for Collaborator due to how it's being used as a verb. Collaborator is capitalized everywhere else, shouldn't it also be capitalized here? Submitted new push.

**Review by marinahand — APPROVED** [review]

**Pull request merged**